david-vazquez/challenges/array-methods/src/find.test.js

```diff
@@ -0,0 +1,30 @@
+function myFind(array, callback) {
+    for(let index=0; index<array.length;index++){
+        if(callback(array[index])) {
+            return array[index];
+        }
+    }
+}
+
+
+describe('Given a myFilter function', () => {
```

Comment:
Change the name of the function!

Reply:
vigila con los nombres! bien visto Laura!